bakerydemo/static/css/main.css

```diff
@@ -702,7 +725,7 @@ span.outline {
 }
 
 .bread-detail figure img {
-  width: auto;
+  height: auto;
```

Comment:
This makes the image enormous on the square-ish breakpoints I usually use. Should probably be defined only for a specific breakpoint.

Before/after:

![mobile-amends-image](https://user-images.githubusercontent.com/877585/34115124-55488c72-e41d-11e7-989c-805e16e8a151.gif)